runtime/ms-rest/lib/webResource.js

```diff
@@ -25,7 +25,6 @@ class WebResource {
     this.url = null;
     this.method = null;
     this.headers = {};
-    this.body = null;
```

Comment:
Instead of removing it. Please set the body to `undefined`.

Reply:
Updated.